desafio11.sql

```diff
@@ -0,0 +1,14 @@
+CREATE VIEW cancoes_premium AS
+SELECT
+  c.nome AS nome,
+  count(*) AS reproducoes
+FROM
+  SpotifyClone.historico_de_reproducoes AS hr
+  INNER JOIN SpotifyClone.cancoes AS c ON hr.cancao_id = c.cancao_id
+  INNER JOIN SpotifyClone.usuarios AS u ON hr.usuario_id = u.usuario_id
+WHERE
+  u.plano_id > 1
```

Comment:
@ficast , tudo bem?
Existe uma recomendação de usar `WHERE` somente se for realmente necessário.
No caso de `JOINS` é possível adicionar outra condição, e as vezes não usar `WHERE`
Usando sua resolução como exemplo, ficaria mais ou menos assim:

```suggestion
  JOIN SpotifyClone.cancoes AS c ON hr.cancao_id = c.cancao_id
  JOIN SpotifyClone.usuarios AS u ON hr.usuario_id = u.usuario_id AND u.plano_id <> 1
```
Na minha resolução, optei por usar `<>` ao invés de `> 1` (também funcionaria :laughing: )
**obs:**  
- Se usar somente `JOIN`, já fica sub entendido que se trata de um `INNER JOIN`, pois é o único que não precisa ser especificado, ao contrario dos `LEFT` e `RIGHT JOIN`.
- Particularmente eu não vejo problema algum em não omitir `ASC`s em `ORDER BY` e não omitir `INNER`s em `JOIN`